app/models/country.rb

```diff
@@ -1,5 +1,16 @@
-class Country < ApplicationRecord
-  has_and_belongs_to_many :sites, class_name: "Site", join_table: 'site_countries'
-  validates :name, :iso3, presence: true
-  validates :name, :iso3, uniqueness: true  
+class Country < ActiveRecord::Base
+  has_and_belongs_to_many :protected_areas
+
+  # has_one :country_statistic
+  # has_one :pame_statistic
+
+  belongs_to :region
+  belongs_to :region_for_index, -> { select('regions.id, regions.name') }, :class_name => 'Region', :foreign_key => 'region_id'
```

Comment:
Why are those `region` and `region_for_index` instead of `country`? I see that names as `country` within the PP repo. So I would replicate that as much as possible

Reply:
Ah sorry, my bad. This is within the country model, so that's correct :)